alphabase/io/tempmmap.py

```diff
@@ -104,12 +121,23 @@ def create_empty_mmap(shape: tuple, dtype: np.dtype, path: str = None, overwrite
     overwrite : bool , optional
         If True the file will be overwritten if it already exists.
         Defaults to False.
+    tmp_dir_name : str, optional
```

Comment:
maybe rename to `temp_dir_abs_path`, and also explain that an absolute path is expected?